packages/main/src/Calendar.js

```diff
@@ -416,11 +468,83 @@ class Calendar extends UI5Element {
 				this._hideMonthPicker();
 			}
 		}
+
+		if (isTabNext(event)) {
+			const header = this.shadowRoot.querySelector("ui5-calendar-header");
+			const monthButton = header.shadowRoot.querySelector("[data-sap-show-picker='Month']");
+			const yearButton = 	header.shadowRoot.querySelector("[data-sap-show-picker='Year']");
```

Comment:
This onKeyDown became too long and hard to read, extract the logic for next and previous in two methods - handleTabNext, handleTabPrevous or handleNext, handlePrevious

Reply:
Done